Update README (#224)
w3c/rdf-tests: README.md

# rdf-tests
Repository for the [RDF Tests Community Group]

To view (HTML) files can be [read directly](https://w3c.github.io/rdf-tests/).

The index.html files are built using a Ruby tool-chain based on the corresponding manifest.ttl files. Update using `rake`. Requires Ruby, rdf-turtle and json-ld gems to be installed.

> NB: as of October 2025, the test suites for RDF and SPARQL are jointly managed the [RDF Tests Community Group] and the [RDF & SPARQL Working Group].

Reviewer: It was from before October 2025 and the active part of the WG runs until April 2027 (unless ...).

Change to "currently".

The situation for external contributors is potentially different (which sign-up agreement?). 
e.g. How decisions get made to accept PRs from external contributors.

```suggestion
Currently, the test suites for RDF ([rdf/](rdf/)) and SPARQL ([sparql/](sparql/)) are jointly managed by the [RDF Tests Community Group] and the [RDF & SPARQL Working Group].
```

> Currently, the test suites for RDF ([rdf/](rdf/)) and SPARQL ([sparql/](sparql/)) are jointly managed by the [RDF Tests Community Group] and the [RDF & SPARQL Working Group].

Reviewer: We can play safe:

```suggestion
Currently, the test suites for RDF (`rdf/`) and SPARQL (`sparql/`) are jointly managed by the [RDF Tests Community Group] and the [RDF & SPARQL Working Group].
```
and maybe put direct links at some point.

The index.html files are built using a Ruby tool-chain based on the corresponding manifest.ttl files. Update using `rake`. Requires Ruby, rdf-turtle and json-ld gems to be installed.

> Currently, the test suites for RDF ([rdf/](rdf/)) and SPARQL ([sparql/](sparql/)) are jointly managed by the [RDF Tests Community Group] and the [RDF & SPARQL Working Group].

Reviewer 2: ```suggestion
Currently, the test suites for RDF ([`rdf/`](rdf/)) and SPARQL ([`sparql/`](sparql/)) are jointly managed by the [RDF Tests Community Group] and the [RDF & SPARQL Working Group].
```

Author: TallTed, are you sure such relative URLs work as expected? I have my doubts about it (especially when the README is displayed on the repo's landing page)...

Reviewer 2: > are you sure such relative URLs work as expected?

The [relative URLs were inserted](https://github.com/w3c/rdf-tests/pull/224/commits/21be05dfa605cd103fa08b94b29a5807ac2272e3#r2394613415) by @afs.

I just tweaked their visual presentation.
